app/src/main/res/layout/insert_wallet_address_view.xml

```diff
@@ -0,0 +1,132 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    >
+
+  <TextView
+      android:layout_width="wrap_content"
+      android:layout_marginBottom="15dp"
+      android:layout_marginLeft="16dp"
+      android:layout_marginStart="16dp"
+      android:layout_marginTop="23dp"
+      android:text="@string/holidayspromotion_title_wallet"
+      android:textColor="@color/black_87_alpha"
+      style="@style/Aptoide.TextView.Regular.L"
+      />
+
+  <TextView
+      android:layout_width="wrap_content"
+      android:layout_marginBottom="15dp"
+      android:layout_marginLeft="16dp"
+      android:layout_marginStart="16dp"
+      android:text="@string/holidayspromotion_message_wallet"
+      android:textColor="@color/black_87_alpha"
+      style="@style/Aptoide.TextView.Regular.M"
```

Comment:
there is already a style with this textColor: @style/Aptoide.TextView.Regular.M.BlackAlpha